docs/source/learning.rst

```diff
@@ -3,9 +3,81 @@
 Learning with spikes
 ------------------------------------------
 
-To be able to learn in spiking neural networks one needs to update the 
-weights between neurons, as in all types of neural networks.
+To be able to learn in spiking neural networks (SNN) one needs to 
+update the weights between neurons, as in all types of neural networks.
+As you probably know, you need smooth, differentiable functions to 
+apply popular algorithms like gradient descent and backpropagation. 
 
-TBD, but for now we can refer to the excellent paper on 
-`Surrogate Gradient Learning in Spiking Neural Networks <https://arxiv.org/abs/1901.09948>`_ by
-Emre O. Neftci, Hesham Mostafa, and Friedemann Zenke.
\ No newline at end of file
+However, spiking neurons do not have smooth activation functions
+(a spike either happens or doesn't).
+This page aims to explain - in an informan manner - how we can train
+SNNs in Norse irregardless of their non-differentiable nature.
+The general approach is described in better detail by
+`Emre O. Neftci, Hesham Mostafa, and Friedemann Zenke <https://arxiv.org/abs/1901.09948>`_.
+
+Before you read further though, make sure you are familiar 
+with how PyTorch 
+`works with Autograd and backpropagation <https://pytorch.org/tutorials/beginner/pytorch_with_examples.html#autograd>`_.
+Norse is basically applying the same principle.
+
+Training Spiking Neural Networks
+=====================================
+
+Many solutions have been attempted to solve this problem with varying 
+success. 
+Here we will only cover the *surrogate gradient* approach, and
+illustrate how  it is implement in norse.
+
+Our approach builds on the SuperSpike method proposed by 
```

Comment:
This is not entirely true. To the best of my knowledge it was first introduced by https://www.pnas.org/content/113/41/11441

Reply:
There is a review by Neftci, Zenke and Mostafa of "Surrogate Gradient methods"